helm-chart/ray-cluster/templates/raycluster-cluster.yaml

```diff
@@ -47,6 +47,51 @@ spec:
 {{ include "ray-cluster.labels" . | indent 10 }}
 
   workerGroupSpecs:
+  {{- range $groupName, $values := .Values.extraWorkers }}
+  {{- if ne $values.disabled true }}
```

Comment:
Can we make sure disabled (or, rather, enabled) is an optional field.
(I'm not sure if it's already optional as is -- I'm not a Helm expert.)

Reply:
It works if disabled/enabled is missing because the condition calls directly on both cases to a missing key 

For example for the head node 
```
.Values.head.disabled
```
will work if disabled is missing.

if disabled was missing, this would not work however
```
.Values.head.disabled.all
```

Reply:
Is it the case that $values.disabled would register a null value if disabled is absent?

Reply:
Yes, yes it would -- I just took a moment to RTFM, don't mind me!